roles/keycloak/defaults/main.yml

```diff
@@ -41,6 +41,9 @@ keycloak_logs_folder: "/var/log/keycloak"
 
 keycloak_logs_max_days: 548  # that's ~18 months
 
+keycloak_logs_level:
```

Comment:
Rename to `keycloak_log_level`

(We should consider using `keycloak_log_*` prefix for all keycloak logging related vars but let's leave this for a future update of the role to avoid breaking existing group vars)